logos/src/logos/main.py

```
@@ -0,0 +1,270 @@
+    with DBManager() as db:
+        return db.connect_process_provider(**data.dict())
+
+
+@app.post("/logosdb/connect_process_model")
+async def connect_process_model(data: ConnectProcessModelRequest):
+    with DBManager() as db:
+        return db.connect_process_model(**data.dict())
+
+
+@app.post("/logosdb/connect_profile_model")
+async def connect_profile_model(data: ConnectProcessModelRequest):
```

Ruff (0.11.9)

`ConnectProcessModelRequest` may be undefined, or defined from star imports

(F405)